src/cgame/cg_snapshot.cpp

```diff
@@ -187,37 +187,34 @@ static void CG_TransitionSnapshot()
 	cg.nextSnap = nullptr;
 
 	// check for playerstate transition events
-	if ( oldFrame )
```

Comment:
for those who (like me) will ask for “why”, the answer was told there:
https://github.com/Unvanquished/Unvanquished/pull/965/files#diff-ca99c9e27a87e660d9e6eef3e81136acL190